net/aria2/files/aria2.init

```diff
@@ -168,16 +168,26 @@ aria2_start() {
 	config_file_tmp="$config_dir/$NAME.conf.tmp"
 	session_file="$config_dir/$NAME.session.$section"
 
-	_make_dir "$config_dir" || {
-		_err "Can't create config dir: $config_dir"
-		return 1
-	}
+	# check directory existence before creating it
+	if [ ! -e "$config_dir" ]; then
```

Comment:
use `-d` ?

Reply:
I notice that you are checking `config_dir` and `session_file` with `-e`, do you mean that `config_dir` can be a link file?

Reply:
My mistake. I will change them to `-d` for `config_dir` and `-f` for `session_file`.

Reply:
Keep it to `2`